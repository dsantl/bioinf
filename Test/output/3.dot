3 0 -10.564192
3 2 23.882426
3 1 23.885577
