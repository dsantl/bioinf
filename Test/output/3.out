3 1 23.885577
3 2 23.882426
3 0 -10.564192
